isisd/isis_spf.c

```diff
@@ -1412,11 +1428,18 @@ static void isis_print_spftree(struct vty *vty, int level,
 	    || !isis_vertex_queue_count(
 		    &area->spftree[tree_id][level - 1]->paths))
 		return;
+	isis = isis_lookup_by_sysid(area->isis->sysid);
```

Comment:
isn't area->isis what you are really looking for?  Why do we need to do a lookup?

Reply:
This is resolved.